include/cinder/gl/TextureFont.h

```diff
@@ -145,7 +145,7 @@ class CI_API TextureFont {
 	bool	isPremultiplied() const { return mFormat.getPremultiply(); }
 
 	//! Returns the default set of characters for a TextureFont, suitable for most English text, including some common ligatures and accented vowels.
-	//! \c "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890().?!,:;'\"&*=+-/\\@#_[]<>%^llflfiphrids����"
+	//! \c "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890().?!,:;'\"&*=+-/\\@#_[]<>%^llflfiphrids����"
```

Comment:
hmm no it wasn't intentional - not actually sure how that might have happened since aside from adding a `CINDER_EMSCRIPTEN` check I don't believe I touched the file. I can re-do that change, looks like I only had to change one spot.